### ISSUE TITLE
sentence does not parse

### ISSUE THREAD

**Issue by tshepang** [stevedonovan/gentle-intro#41]
In 7-shared-and-networking.md, I struggle to understand this:

> However, Cell only works with values that can be copied, that is, they implement Copy, like primitive types and structs containing them marked as 'derive(Copy)'.

It seems to say that Cell works with structs so long as its members are copy, and itself is marked as Copy.

As an aside, perhaps splitting the sentence in 2 would help as well... it feels overlong.

**Comment by stevedonovan** [comment]
Yep, definitely too long. You did extract the meaning I was trying to put into the sentence, but the sentence is clumsy.

**Comment by stevedonovan** [comment]
How does this sound?

     However, `Cell` only works with values that can be copied 
     like primitive types and user types deriving the `Copy` trait.

**Comment by tshepang** [comment]
I would:
> However, `Cell` only works with `Copy` types (e.g. primitive types and user-defined types deriving the `Copy` trait).
